extensions/firebase-import-export/src/firebase-export.tsx

```diff
@@ -0,0 +1,48 @@
+import { getPreferenceValues, showHUD, closeMainWindow } from "@raycast/api";
+import { backups, initializeFirebaseApp } from "firestore-export-import";
+import { IExportOptions } from "firestore-export-import/dist/helper";
+import moment from "moment";
+import { convertStringToNumber, readFile, writeFile } from "./utils";
+import { Preferences } from "./interfaces";
+
+/* Getting the preferences from the Raycast API. */
```

Comment:
I'd argue that most of the comments in the file add more noise as the API is already pretty clear on what it does, so you could remove them.